_docs/_latest/configure/general/non-org-root.md

```diff
@@ -28,57 +28,38 @@ manually assign the correct roles later.
-to the target folder:
-`folders/<foo_folder_id>`.
+1. Edit `main.tf` and set `composite_root_resources` variable to the target 
+folder: `["folders/<foo_folder_id>"]`.
 
-1. **NEW for version 2.12.0+**: You can use the `composite_root_resources`
-   configuration to include multiple resources in a single Forseti installation.
-   See [Configure Inventory]({% link _docs/latest/configure/inventory/index.md %})
+1. You can use the `composite_root_resources` configuration to include 
+   multiple resources in a single Forseti installation. See [Configure Inventory]({% link _docs/latest/configure/inventory/index.md %})
    for more details.
 
-1. If Forseti was installed with Org Admin credentials, then the org-level
-   roles will be inherited on the folder-level.
-
-1. If Foresti was not installed with Org Admin credentials, then you need
-   to grant the Forseti server service account to have the same roles on the
-   target resources, as was [originally granted on the
-   organization]({% link _docs/latest/concepts/service-accounts.md %}#the-server-service-account).
-
 1. Saving changes.
-   1. Save the changes to `forseti_conf_server.yaml` file.
-   1. Upload `forseti_conf_server.yaml` [to GCS bucket]({% link _docs/latest/configure/general/index.md %}#moving-configuration-to-cloud-storage).
-
-1. Use the updated configuration.
-   1. SSH to the Forseti server VM.
-   1. Use sudo gsutil to copy the `forseti_conf_server.yaml` file from GCS
-   bucket to `/home/ubuntu/forseti-security/configs/`.
-   1. Make the server [reload the updated configuration]({% link _docs/latest/use/cli/server.md %}).
-
-1. Create a [custom role](https://cloud.google.com/iam/docs/creating-custom-roles) with `storage.buckets.get` permission 
-and assign it to your Forseti server service account.
+   1. Save the changes to `main.tf` file.
+   1. Run command `terraform plan` to see the infrastructure plan. 
+   1. Run command `terraform apply` to apply the infrastructure build.
+   
+When you run Forseti again, all the resources from the target root
+will be collected in Inventory and audited.
 
 ## Configure Forseti to Run on Projects
 
-**NEW for version 2.12.0+**: As an alternative, you can use the
-`composite_root_resources` configuration to include multiple resources in a
-single Forseti installation.
+As an alternative, you can use the `composite_root_resources` configuration to 
+include one or more resources from GCP resource hierarchy in a single Forseti 
+installation.
 See [Configure Inventory]({% link _docs/latest/configure/inventory/index.md %})
 for more details.
 
-1. This assumes that Forseti is not installed with Org Admin credential, and
-you want Forseti to run on projects that you own. If Forseti is installed
-with Org Admin credential, then all the resources in the organization
-will be returned.
+For example: `composite_root_resources:` `["folders/<foo_folder1_id>", "projects/<foo_project1_id"]`
```

Comment:
nit: we should not use folder as example here

Reply:
Updated